src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -26,14 +26,14 @@ interface ConversationRouteProps {
 
 const ConstrainedHeightFlex = styled(Flex)`
   height: calc(100vh - 145px);
-  ${media.xs`
+  ${media.sm`
     height: calc(100vh - 55px);
   `}
   & > * {
     overflow-y: scroll;
     overflow-x: hidden;
   }
-  & > .fresnel-greaterThan-xs {
+  > .fresnel-greaterThan-sm {
```

Comment:
Adjustment needed to align inbox header with inbox col:

Before:
<img width="1108" alt="Screen Shot 2020-07-15 at 4 28 14 PM" src="https://user-images.githubusercontent.com/12748344/88084808-f0724980-cb52-11ea-8728-ac7151efa6d7.png">

<br>
<br>

After:
<img width="1642" alt="Screen Shot 2020-07-15 at 4 28 06 PM" src="https://user-images.githubusercontent.com/12748344/88084766-df293d00-cb52-11ea-935f-1f0604b596b1.png">